operations/commit_queue.go

```diff
@@ -274,6 +278,87 @@ func enqueuePatch() cli.Command {
 	}
 }
 
+func backport() cli.Command {
+	return cli.Command{
+		Name:  "backport",
+		Usage: "automatically backport low-risk commits",
+		Flags: mergeFlagSlices(
+			addPatchFinalizeFlag(),
+			addVariantsFlag(),
+			addTasksFlag(),
+			addProjectFlag(),
+			addPatchAliasFlag(),
+			addPatchBrowseFlag(
+				cli.StringFlag{
+					Name:  joinFlagNames(existingPatchFlag, "e"),
+					Usage: "existing commit queue patch",
+				},
+			)),
+		Before: mergeBeforeFuncs(
+			setPlainLogger,
+			requireProjectFlag,
+			requireStringFlag(existingPatchFlag),
+		),
+		Action: func(c *cli.Context) error {
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
+
+			confPath := c.Parent().Parent().String(confFlagName)
+			patchParams := &patchParams{
+				Tasks:    c.StringSlice(tasksFlagName),
+				Variants: c.StringSlice(variantsFlagName),
+				Alias:    c.String(patchAliasFlagName),
+				Finalize: c.Bool(patchFinalizeFlagName),
+				Backport: c.String(existingPatchFlag),
```

Comment:
I think it's kind of confusing that this is called Backport, because I keep thinking of it as "this is the patch that is doing the backport" rather than "this is the patch to backport." I don't really have a solution though, besides ToBackport. Perhaps otherwise just some comments where the field Backport is initialized?

Reply:
I renamed it `BackportOf`